docs/2-tutorials/2-develop/6-push-notifications.html.md

```diff
@@ -2,7 +2,164 @@
+
+5. `cd` into the ***www*** folder and look around at the files and subfolders in there, this is the content of your app, with the entry point being the index.html file.
+
+		$ cd www/
+
+ <div class="alert--tip">**TIP:** Details about the rest of the files and folders created in the root project  will be covered in guides further along. For now just focus on the ***www*** folder and its contents.</div>
+
+6. Type `$ phonegap serve`. You will receive the server address the app is being hosted on in the output received in the console (`192.168.1.11:3000` in this example):
+
+		$ phonegap serve
+		[phonegap] starting app server...
+		[phonegap] listening on 192.168.1.11:3000
+		[phonegap]
+		[phonegap] ctrl-c to stop the server
+		[phonegap]
+
+7. Now go to your mobile device where the PhoneGap Developer App is running, enter the server address on the main screen and tap **Connect**.
+
+  <img class="mobile-image" src="/images/dev-app-enter-add.png" alt="PhoneGap Developer App, iOS"/>
+
+
+ <div class="alert--info"> **NOTE:** Tap directly on the server address displayed in the terminal screen of the PhoneGap Developer app to change it to match yours. The value filled in by default is only a sample. </div>
+
+ You should see the connection occur followed by a success message as shown below. If you receive an error of any kind, ensure
+ once again that you are connected to the same network on both your  and computer. You could also check the
+ [issue tracker](https://github.com/phonegap/phonegap-app-developer/issues) and [PhoneGap Google Groups](https://groups.google.com/forum/#!forum/phonegap)
+ list for further help.
+
+  <img class="mobile-image" src="/images/dev-app-success.jpg" alt="Developer App, connection success"/>
+
+
+ Once the PhoneGap Developer app connects and loads your mobile application, it should be displayed for preview as shown below:
+
+ <img class="mobile-image" src="/images/dev-app-preview.jpg" alt="Developer App, preview"/>
+
+ The grey `Requesting Registration ID` line under the PhongGap Bot should turn into a green `Registered` line and you should see the following in your terminal session.
+
+     [phonegap] [console.log] registration event: APA91bE1MmeTc92igNoi5OkDWUV
+
+ <div class="alert--warning">Obviously, your registration ID will be different but you get the picture.</div>
+
+### Sending Your First Push
+
+1. Open a new terminal window and enter the following command if you are testing on iOS:
+
+		$ phonegap push --deviceID APA91bE1MmeTc92igNoi5OkDWUV --service apns --payload '{ "aps": { "alert": "Hello World" } }'
+
+   or enter the following command if you are testing on Android:
+
+		$ phonegap push --deviceID APA91bE1MmeTc92igNoi5OkDWUV --service gcm --payload '{ "data": { "title": "Hello", "message": "World"} }'
+
+   replacing the deviceID with the one you received from the registration event in the previous step.
+
+2. This will result in a push notification being received by the app.
```

Comment:
This is up to you but it may be nice to mention, if you switch your app to the background or lock screen and run the same command and see these views (because they illustrate push so nicely). 

![image](https://cloud.githubusercontent.com/assets/359935/14828075/517415d8-0bb3-11e6-8e9a-f9a21632e2d2.png)
![image](https://cloud.githubusercontent.com/assets/359935/14828096/6a4d0844-0bb3-11e6-8128-8034c1526015.png)

And maybe add some screenshots for the Android side of things. 

(On a side note for docs in general - If you want to do any kind of side by side images, it's best to put them in one single image yourself first using Photoshop or an image tool so they're laid out better, we need some new styles for this situation.)